### ISSUE TITLE
New TranslateParser not exported?

### ISSUE THREAD

**Issue by biesbjerg** [ocombe/ng2-translate#346]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => check the FAQ and search github for a similar issue or PR before submitting
[ ] support request => check the FAQ and search github for a similar issue before submitting
[ ] feature request
```

**Current behavior**
I'm not totally sure what's going on, since that looking at the code it would seem the just pushed `TranslateParser` / `DefaultParser` is indeed exported, I'm not able to import in my code.

![image](https://cloud.githubusercontent.com/assets/230182/20893403/ed5cd5da-bb11-11e6-979d-0c5f0da21b5e.png)

As you can see, I'm able to see the new `TranslateDirective`, so I'm positive that I'm on v4.1.0.

Any ideas?
* **ng2-translate version:** 4.1.0

* **Angular version:** 2.2.1
 
* **Language:** Typescript 2.0.10

**Comment by biesbjerg** [comment]
Huh, it seems it is exported as just `Parser` and no (apparant) way to import `DefaultParser`.

Is that intentional? I'd like to extend `DefaultParser` to change `templateMatcher` regexp.

**Comment by ocombe** [comment]
It's not intentional, I'll fix this ! Thanks for the report

**Comment by ocombe** [comment]
Actually this is already fixed, waiting on the last changes in #332 before I release a new version

**Comment by ocombe** [comment]
I will release a new version now, and I'll release a new one later with #336

**Comment by biesbjerg** [comment]
Nice :-) And awesome that we'll be able to customize parser - That should solve my issue with wanting to use 'This is a { value }' message keys in templates :-)

**Comment by ocombe** [comment]
Yes, it's available in 4.2.0
